src/main/java/mate/academy/service/UserService.java

```diff
@@ -15,6 +15,15 @@
      * Return <code>null</code> if there is no suitable user
      */
     public User findByEmail(String email) {
-        return null;
+        if (email == null) {
+            return null;
+        } else {
+            for (User user : users) {
+                if (user.getEmail().equals(email)) {
+                    return user;
+                }
+            }
+            return null;
+        }
```

Comment:
```suggestion
		if (email != null) {
	        for (User user : users) {
	            if (user.getEmail().equals(email)) {
	                return user;
	            }
	        }
	    }
        return null;
```
try not to complicate if-else construction